<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductForm</class>
 <widget class="QDialog" name="AddProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PRODUCT NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productNameLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>QUANTITY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PRODUCT PRICE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productPriceLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="productQuantityLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
